client/src/components/FilterChip.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { FormikHandlers } from 'formik';
+
+type HandleFormikChange = FormikHandlers['handleChange'];
+
+type HandleClick = () => void;
+
+type FilterChipProps = {
+  name?: string;
+  text: string;
+  isActive: boolean;
+  handleOnClick: HandleFormikChange | HandleClick;
+};
+
+const FilterChip: React.FC<FilterChipProps> = ({ text, isActive, handleOnClick }) => {
+  return (
+    <button className={`filter-base filter-chip ${isActive && 'active'}`} onClick={handleOnClick}>
```

Comment:
Remember that `&&` will return false and we're appending a false in the `className` we should use ternaries.

Code example:
```jsx
 <button className={`filter-base filter-chip ${isActive ? 'active' : ''}`} onClick={handleOnClick}>
```